### PR TITLE
Add support for Emacs Lisp

### PR DESCRIPTION
**Pull request by raxod502**
We use IELM.

### CODE REVIEW

**Comment by raxod502** [comment]
Now ready for review and merge.

**Comment by raxod502** [comment]
I don't understand why the build is failing. It passes locally.

**Comment by raxod502** [comment]
The build passed after I restarted it twice.

File: languages/elisp/main.go [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+func Execute(config *utils.Config) {
+
+	emacs, err1 := exec.LookPath("emacs")
+
+	if err1 != nil {
+		panic(err1)
+	}
+
+	execPath, err2 := os.Executable()
```
Comment (turbio) [inline languages/elisp/main.go]: just keep modifying the same err, `:=` will set any existing variables

Comment (raxod502) [reply languages/elisp/main.go]: Fixed, thanks.

File: languages/elisp/main.go [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+	os.Setenv("PRYBAR_PS1", config.Ps1)
+
+	if config.Quiet {
+		os.Setenv("PRYBAR_QUIET", "1")
+	} else {
+		os.Setenv("PRYBAR_QUIET", "")
+	}
+
+	if config.Args != nil {
+		os.Setenv("PRYBAR_FILES", strings.Join(config.Args, "\000"))
```
[the remark this answers is not in the document]
Comment (raxod502) [reply languages/elisp/main.go]: Changed.

**Pull request merged**